tests/testthat/_snaps/harmonic.md

```diff
@@ -5,7 +5,9 @@
         frequency = 1, cycle_size = NA)
     Condition
       Error in `step_harmonic()`:
-      ! cycle_size must have at least one non-NA numeric value.
+      x `cycle_size` must have at least one non-NA numeric value.
+      i It was `NA`.
```

Comment:
Wondering whether a different markup here could supply a more informative message--wishing this read something like "It was a single missing value." Also, "Only missing values were present" feels redundant in this case.

Reply:
I think the last part is redundant. So i'll remove it